examples/experimental/terrain/README.md

```diff
@@ -0,0 +1,30 @@
+This is a minimal standalone version of the TerrainLayer example
+on [deck.gl](http://deck.gl) website.
+
+### Usage
+
+Copy the content of this folder to your project. 
+
+To load the terrain tiles, you need a [Mapbox access token](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/). You can either set an environment variable:	
+
+```bash	
+export MapboxAccessToken=<mapbox_access_token>	
+```	
+
+Or set `MAPBOX_TOKEN` directly in `app.js`.
+
+```bash
+# install dependencies
+npm install
+# or
+yarn
+# bundle and serve the app with webpack
+npm start
+```
+
+### Data format
+
+Mapbox's [terrain API](https://docs.mapbox.com/help/troubleshooting/access-elevation-data/#mapbox-terrain-rgb) encodes elevation data in raster tiles.
+
+To use other data sources, check out
+the [documentation of TerrainLayer](../../../docs/api-reference/geo-layers/terrain-layer.md).
```

Comment:
Ideally we would move some of this information into loaders.gl `TerrainLoader` docs.

Reply:
That link will not work in loaders.gl

Reply:
A we going to publish terrain on the website? If not, I would fix this link to "https://deck.gl/docs/api-reference/geo-layers/terrain-layer"

Reply:
Sure let's do that for now. It could be nice to publish on the website if we can clean it up, but for now we can keep it in experimental folder and try to gradually get it ready.